src/churn/churn.ts

```diff
@@ -288,6 +315,17 @@ export var filterCandidatesFromSdp = (sdp:string) : string => {
       this.havePipe_();
     }
 
+    private makeSampleTargetLengthDistribution_ = () : Array<number> => {
+      var result : Array<number> =[];
```

Comment:
Style guide: types have colons right next to them.  So ":Array<number>" not ": Array<number>."  Not that you were claiming to adhere to the guide, but FYI.